javascript/node/selenium-webdriver/test/testing/index_test.js

```diff
@@ -105,3 +105,55 @@ describe('Mocha Integration', function() {
     });
   });
 });
+
+describe('Mocha async "done" support', function() {
```

Comment:
Would you mind adding tests for ControlFlow integration? Something like:

    var flow, order;
    before(function() {
      order = [];
      flow = promise.controlFlow();
      flow.execute(function() { order.push(1); });
    });

    test.it('', function(done) {
      flow.execute(function() { order.push(2); });
      flow.execute(function() { order.push(3); }).then(function() {
        order.push(4);
      });
      done();
    })

    after(function() {
      assert.deepEqual([1, 2, 3, 4], order);
    })